From 4dc15a56017633e5e06ba345da188d773c6a9b16 Mon Sep 17 00:00:00 2001
From: Vincent Abriou <vincent.abriou@st.com>
Date: Tue, 20 Sep 2016 16:15:32 +0200
Subject: [PATCH 12/13] [PATCH] WIP allocators: define
 GST_CAPS_FEATURE_MEMORY_DMABUF

This patch comes from Julien Isorce as part of the VAAPI updates
for dmabuf support.
Details : https://bugzilla.gnome.org/show_bug.cgi?id=759358

Change-Id: Id5fdd8e049af7a363c25fbecbc9f878faeba6b46
Reviewed-on: https://gerrit.st.com/58101
Reviewed-by: Vincent ABRIOU <vincent.abriou@st.com>
Tested-by: Vincent ABRIOU <vincent.abriou@st.com>
Reviewed-by: Fabien DESSENNE <fabien.dessenne@st.com>
Tested-by: Fabien DESSENNE <fabien.dessenne@st.com>
---
 gst-libs/gst/allocators/gstdmabuf.h | 12 ++++++++++++
 1 file changed, 12 insertions(+)

diff --git a/gst-libs/gst/allocators/gstdmabuf.h b/gst-libs/gst/allocators/gstdmabuf.h
index c09d2f1..913a65c 100644
--- a/gst-libs/gst/allocators/gstdmabuf.h
+++ b/gst-libs/gst/allocators/gstdmabuf.h
@@ -25,6 +25,18 @@
 
 G_BEGIN_DECLS
 
+/**
+ * GST_CAPS_FEATURE_MEMORY_DMABUF: (value "memory:DMABuf") (type const gchar *)
+ *
+ * Constant that defines the caps feature name for dmabuf memory.
+ *
+ * It helps pipeline negotiation when the exporter is the upstream element from
+ * the importer point of view.
+ *
+ * Since: 1.8
+ */
+#define GST_CAPS_FEATURE_MEMORY_DMABUF "memory:DMABuf"
+
 #define GST_ALLOCATOR_DMABUF "dmabuf"
 
 GstAllocator * gst_dmabuf_allocator_new (void);
-- 
2.7.4

